unique_sdk/unique_sdk/api_resources/_message.py

```diff
@@ -88,7 +88,32 @@ def list(
         return result
 
     @classmethod
-    def retrieve(
+    async def list_async(
```

Comment:
maybe also rename to be explicit.

Reply:
not sure what you mean here? The names are the same for sync and async just that async have this suffix _async

Reply:
ah. perfect, so its backwards compatible. nice